tools/run_tests/xds_k8s_test_driver/tests/failover_test.py

```diff
@@ -70,14 +70,14 @@ def test_failover(self) -> None:
         with self.subTest('04_create_forwarding_rule'):
             self.td.create_forwarding_rule(self.server_xds_port)
 
+        default_test_servers: List[_XdsTestServer] = []
+        alternate_test_servers: List[_XdsTestServer] = []
```

Comment:
I don't think it's required to assign the value, you should be able to just specify the type.
```suggestion
        default_test_servers: List[_XdsTestServer]
        alternate_test_servers: List[_XdsTestServer]
```

Reply:
From what I see, declaring the type doesn't necessary assign it to None:

```python3
Python 3.10.4 (main, Mar 25 2022, 04:37:13) [Clang 13.0.0 (clang-1300.0.29.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from typing import List

# not defined - error
>>> print(default_test_servers)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'default_test_servers' is not defined

# only type declared - same as not defined - error
>>> default_test_servers: List[str]
>>> print(default_test_servers)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'default_test_servers' is not defined

# type and assignment - all fine
>>> default_test_servers: List[str] = []
>>> print(default_test_servers)
[]
```

Reply:
Confirming py3.6 behaves the same as above.

However, if mypy complains about something and it's easier to assign the value - it's fine by me.

Reply:
TIL it will end up with "not defined" instead of giving it a value... This is very different than other languages. I will update all test cases to use the suggested pattern.